spec/amp-cors-requests.md

```diff
@@ -53,7 +53,12 @@ The resulting HTTP response has to also contain the following headers:
 #### Note on non-idempotent Requests
 When making CORS requests that would change the state of your system (e.g. user subscribes to or unsubscribes from a mailing list) the first two steps you need to make sure to do:
 
-1. Check the `Origin` header. If the origin was not `*.ampproject.org` or the publisher's origin, stop and return an error response.
-2. Check the `__amp_source_origin` query parameter. If it's not the publisher's origin stop and return an error response.
+1. Check if the request has `AMP-Same-Origin: true` header. If yes, proceed to process the request safely (skip next steps).
```

Comment:
Another independent thing is that I think it would be better to move this point into the main "CORS Security in AMP" point and call it the third component besides Origin and __amp_source_origin.